styles.css

```diff
@@ -768,4 +833,50 @@ textarea {
   .icon-container-3 {
     display: none;
   }
+
+  .mobile-menu {
+    position: absolute;
+    left: -107%;
+    top: 0;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 1rem;
+    background-color: white;
+    width: 100%;
+    height: 100%;
+    margin-top: 75px;
+  }

```

Comment:
- [OPTIONAL] In order to provide a better User Experience (UX) it would be better if when the menu is open even if we scroll down we will only keep seeing the only menu and the given options but on your website when the menu is opened and we scroll down we can be able to see the other part of the page. Kindly fix that to prove the best User Experience.

     - ### WEBSITE BEHAVIOUR
    ![chrome-capture-2023-3-18](https://user-images.githubusercontent.com/87197412/232711698-fd11aad1-67aa-4f0c-92ed-5fac7a38566d.gif)